qiskit/qasm3/exporter.py

```diff
@@ -448,15 +449,33 @@ def build_quantumdeclarations(self):
         )
         # aliases
         for qreg in self.circuit_ctx[-1].qregs:
-            qubits = []
+            elements = []
+            # Greedily consolidate runs of qubits into ranges.  We don't bother trying to handle
+            # steps; there's no need in generated code.  Even single qubits are referenced as a
```

Comment:
Is there a missing word at the end of this line?

Reply:
Not even a full word, but 5/6 of one!  Thanks - fixed.